Source/common/SNTBlockMessage.m

```diff
@@ -109,7 +109,12 @@ + (NSString *)stringFromHTML:(NSString *)html {
   return [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
 }
 
-+ (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event {
++ (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event customURL:(NSString *)url {
+  if (url.length) {
+    if ([url isEqualToString:@"null"]) return nil;
+    return [NSURL URLWithString:url];
```

Comment:
If an invalid URL is passed I think we should log here.

Reply:
Done